package.json

```diff
@@ -17,7 +17,7 @@
   },
   "private": true,
   "dependencies": {
-    "@aion-dk/js-client": "^0.1.11",
+    "@aion-dk/js-client": "^0.1.13",
```

Comment:
If we want to manually manage patch upgrades like this (which I think makes sense), can we disable optimistic upgrades?

Reply:
Yes, that makes complete sense. I'll update to "0.1.13"

Reply:
Cool, thanks! Curious, what was the driving force for this upgrade? I don't see any corresponding code changes here taking advantage of the new version.

Reply:
You are right. Apologies for my lazy PR description 😬 I updated the description to reflect the reasoning.